olive/passes/onnx/vitis_ai/quant_utils.py

```diff
@@ -238,31 +238,42 @@ def compute_scale_zp_pof2s(rmin, rmax, qmin, qmax, symmetric=False):
     # Adjust rmin and rmax such that 0 is included in the range. This is
     # required to make sure zero can be represented by the quantization data
     # type (i.e. to make sure qmin <= zero_point <= qmax)
-    rmin = min(rmin, 0)
-    rmax = max(rmax, 0)
+    rmin = np.minimum(rmin, np.array(0, dtype=rmin.dtype))
```

Comment:
it seems the rmin need also check the dtype. https://aiinfra.visualstudio.com/PublicPackages/_build/results?buildId=398291&view=logs&j=9e2040b5-c08e-54c3-cba8-c40a938d4cef&t=621d66f3-03a9-55d1-fbc2-2879c9c8c732

![image](https://github.com/microsoft/Olive/assets/817030/0bd3773e-76cf-4524-9886-e8b8cb0b89d7)